modules/developer-lightspeed/proc-installing-and-configuring-lightspeed.adoc

```diff
@@ -103,27 +97,21 @@ data:
   app-config.yaml: |-
     backend:
       csp:
-         upgrade-insecure-requests: false
-       img-src:
+        upgrade-insecure-requests: false
+        img-src:
           - "'self'"
           - "data:"
           - https://img.freepik.com
           - https://cdn.dribbble.com
           - https://avatars.githubusercontent.com # This is to load GitHub avatars in the UI
-       script-src:
-           - "'self'"
-           - https://cdn.jsdelivr.net
+        script-src:
+          - "'self'"
+          - https://cdn.jsdelivr.net
 
     lightspeed:
-      # REQUIRED: Configure LLM servers with OpenAI API compatibility
-      servers:
-        - id: ${LLM_SERVER_ID}
-          url: ${LLM_SERVER_URL}
-          token: ${LLM_SERVER_TOKEN}
-
-      # OPTIONAL: Enable/disable question validation (default: true)
-      # When enabled, restricts questions to RHDH-related topics for better security
-      questionValidation: true
+      mcpServers:
+      - name: mcp::backstage
+        token: ${MCP_TOKEN}
```

Comment:
Since we are defining MCP_TOKEN, it will be nice to add [guide](https://backstage.io/docs/auth/service-to-service-auth/#static-tokens) to generate this external static token.


We have to do the following to inject the value to the RHDH deployment.

1. we need to create a secret for this called `lightspeed-secrets` 
```
apiVersion: v1
kind: Secret
metadata:
  name: lightspeed-secrets
  namespace: <_your_namespace_>
type: Opaque
stringData:
  MCP_TOKEN: <_your_static_token>
```

2. we need to mount it to the backstage CR, like below
```yaml
spec:
  application:
    extraEnvs:
      secrets:
        - name: lightspeed-secrets

````